makelib/k8s_tools.mk

```diff
@@ -38,7 +38,7 @@ KUSTOMIZE := $(TOOLS_HOST_DIR)/kustomize-$(KUSTOMIZE_VERSION)
 
 # the version of helm 3 to use
 USE_HELM3 ?= false
-HELM3_VERSION := v3.2.4
+HELM3_VERSION ?= v3.4.0
```

Comment:
yes this is definitely better to allow this value to already be set to the callers desired version, thanks for making it.

what do you think about setting the default to v3.4.2, the latest patch version for the v3.4 release?
https://github.com/helm/helm/releases/tag/v3.4.2

Reply:
+1. Submitted the other commit. Do you think it's necessary to set helm version 2 to the version above v2.17.0 too, the latest one?

Reply:
thanks! no i don't think it's necessary to also update version 2 at the same time. this PR is focused (and has a distinct need for updating the version of) on version 3 so we can just keep that focus.